docs/variants/pc_engines/initial-deployment.md

```diff
@@ -0,0 +1,42 @@
+# Initial deployment
+
+Initial flashing of Dasharo firmware can be done from Linux using flashrom with
+the internal programmer.
+
+## Deploy using Dasharo Tools Suite
+
+For simplicity we recommend using
+[Dasharo Tools Suite](../../dasharo-tools-suite/overview.md) to
+omit all compilation steps and deploy the Dasharo firmware seamlessly.
+
+1. Boot Dasharo Tools Suite.
+2. Perform Dasharo installation.
+
+After the successful operation, DTS will reboot the platform.
+
+This concludes Dasharo deployment process using DTS.
+
+## Build flashrom
+
+Please follow generic guide for [Dasharo flashrom fork](../../osf-trivia-list/deployment.md#how-to-install-dasharo-flashrom-fork).
+
+## Reading flash contents
+
+Always prepare a backup of the current firmware image. To read from the flash
+and save it to a file (`dump.rom`), execute the following command:
+
+```bash
+flashrom -p internal -r dump.rom
+```
+
+## Flashing Dasharo
+
+To flash Dasharo on the platform, execute the following command - replace
+`<variant>` with the APU variant (2, 3, 4 or 6) and `<version>` with the
+Dasharo image version, e.g. `v0.9.0`.
```

Comment:
```suggestion
Dasharo image version, e.g. `v0.9.0`, `v24.02.01.01`,`v4.0.34`.
```

Reply:
`v4.0.34` ? Legacy? I thought we will not be keeping legacy branch on Dasharo coreboot. Is there any reason to do so?

Reply:
https://github.com/Dasharo/docs/pull/794/commits/e59c85b042692ab32d4c97c05a3c2367d9eb1534

Reply:
@miczyg1 The reason for v4.0.34 is to preserve the value of old releases, but we would have to change the methodology of fork maintenance to make that release. As discussed with Andrew on matrix and mentioned [here](https://github.com/Dasharo/dasharo-issues/issues/310)